tensorboard-controller/rockcraft.yaml

```diff
@@ -0,0 +1,60 @@
+# Source https://github.com/kubeflow/kubeflow/blob/v1.9.0/components/tensorboard-controller/Dockerfile
+name: tensorboard-controller
+summary: An image for tensorboard controller
+description: |
+  This image is used as part of the Charmed Kubeflow product. The controller needs 
+  for providing the measurements and visualizations needed during the machine learning workflow.
+version: "1.9.0"
+license: Apache-2.0
+base: ubuntu@22.04
+services:
+  tensorboard-controller:
+    override: replace
+    summary: "tensorboard-controller service"
+    startup: enabled
+    user: ubuntu
+    command: "/manager"
+platforms:
+  amd64:
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+       dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > \
+       ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  tensorboard-controller:
+    plugin: go
+    source: https://github.com/kubeflow/kubeflow
+    source-type: git
+    source-tag: v1.9.0
+    source-subdir: components/tensorboard-controller
+    build-snaps:
+      - go/1.17/stable
+    build-packages:
+      - apt
+      - bash
+    build-environment:
+      - CGO_ENABLED: 0
+      - GOOS: linux
+    stage-packages:
+      - bash
+    override-build: |
+      cd $CRAFT_PART_SRC_WORK
```

Comment:
Can we also remove the `override-build` similarly to the profile controller rock?

Reply:
@kimwnasptd Fixed